lib/l4p/tle_udp.h

```diff
@@ -98,6 +98,19 @@ int
 tle_udp_stream_get_param(const struct tle_stream *s,
 	struct tle_udp_stream_param *prm);
 
+/**
+ * Get current UDP stream RX queue size
+ * @param us
+ *   Stream to retrieve state information from.
+ * @return
+ *   On success, the value will be >= 0 and can safely be stored
+ *   in an uint32_t. This value will be the RX queue size of the
+ *   stream.
+ *   On error, the value will be < 0 and be one of the following:
```

Comment:
You say "one of the following" yet only list one possible value...I'd revise this for clarity.

Reply:
...there are a number of functions that bubble up error codes that are not actually captured in the doxy like comments.  its one of my biggest gripes about this.  there are also a number of places where it expands to something like `-(-err_code)` but says errors will be negative...

Reply:
Haha, damn programmers.  :^)  I reckon that it is what it is on this one then.